### PR TITLE
chore(style): create a cheat sheet for the style guide

### PR DESCRIPTION
**Pull request by ghost**
This PR includes adding a new high-level page that quickly references our most important style guide topics, based on actual usage. I wanted to brand it as a cheat sheet for writers and as a supplementary reference guide to our left nav (also in lieu of having a working search functionality for style guide topics). 

Jira: NR-288156

### CODE REVIEW

**Comment by ally-sassman** [comment]
@jeff-colucci @paperclypse  I'd love your thoughts on which pages are here, and if anything about the structure should change.

**Comment by paperclypse** [comment]
@ally-sassman very cool! thank you!

File: src/content/docs/style-guide/cheat-sheet.mdx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      <td>Hero-ing</td>
+      <td>Information about GitHub processes, merging, and the role of a Docs hero.</td>
+      <td>
+        
+        * [What is a Docs hero?](/docs/agile-handbook/heroing/what-is-a-hero/)
+        * [GitHub intro](/docs/style-guide/writing-docs/writer-workflow/github-intro/)
+
+      </td>
+    </tr>
+  </tbody>
```
Comment (paperclypse) [inline src/content/docs/style-guide/cheat-sheet.mdx:80]: @ally-sassman looks like you need a closing table tag here

**Review by paperclypse — DISMISSED** [review]
A good list! Thanks for putting this together!

File: src/content/docs/style-guide/cheat-sheet.mdx (hunk 1 above)
Comment (ally-sassman) [inline src/content/docs/style-guide/cheat-sheet.mdx:80]:
```suggestion
  </tbody>
  </table>
```

**Review by paperclypse — APPROVED** [review]
Looks good!

**Pull request merged**